src/components/sections/Incentives/IncentivesHeader.tsx

```diff
@@ -1,37 +1,19 @@
-import Icon from 'src/components/ui/Icon'
-
 import Incentives from './Incentives'
 import Section from '../Section'
 
-const incentives = [
-  {
-    icon: <Icon name="Truck" width={32} height={32} />,
-    title: 'Buy online',
-    firstLineText: 'Get Free Shipping',
-  },
-  {
-    icon: <Icon name="Calendar" width={32} height={32} />,
-    title: 'Free return',
-    firstLineText: '30 days to return',
-  },
-  {
-    icon: <Icon name="Gift" width={32} height={32} />,
-    title: 'Gift cards',
-    firstLineText: '$20 / $30 / $50',
-  },
-  {
-    icon: <Icon name="Storefront" width={32} height={32} />,
-    title: 'Physical Stores',
-    firstLineText: '+40 Stores in Brazil',
-  },
-  {
-    icon: <Icon name="ShieldCheck" width={32} height={32} />,
-    title: 'Buy online',
-    firstLineText: 'Get Free Shipping',
-  },
-]
+interface Incentive {
+  icon: string
+  title?: string
+  firstLineText: string
+  secondLineText?: string
```

Comment:
Just noticed now the CMS version didn't seem to have this `secondLineText` prop described in the [`cms/sections.json`](https://github.com/vtex-sites/storeframework.store/pull/47/files#diff-15e0a47bc03c12a18265b61d558b96969b4de3fefc7ad6d020b327049ef33531). Why is that?

Reply:
I didn't pay attention on that one when I was updating the json, going to add that now. 
Thanks Wands!